src/index.html

```diff
@@ -5,10 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Montserrat:wght@400;600&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li">
+            <a class="text is-active" href="#">
+              Apple
+            </a>
+          </li>
+          <li class="li">
+            <a class="text" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="li">
```

Comment:
Add a blank line between the links
```suggestion
          </li>
          
          <li class="li">
```